actors/multisig/src/lib.rs

```diff
@@ -569,5 +585,6 @@ impl ActorCode for Actor {
       ChangeNumApprovalsThreshold => change_num_approvals_threshold,
       LockBalance => lock_balance,
       UniversalReceiverHook => universal_receiver_hook,
```

Comment:
We _could_ remove this, since the Universal Receiver Hook's method is in the exported FRC-42 range, so it gets covered by the fallback. But I'd rather keep this explicit branch for clarity. IMO, the account actor should do the same (which it doesn't).